source/wp-content/themes/wporg-developer-2023/src/code-methods/block.php

```diff
@@ -35,7 +35,7 @@ function render( $attributes, $content, $block ) {
 		return '';
 	}
 
-	$content = get_methods_content( $block->context['postId'] );
+	$content = get_methods_content();
```

Comment:
Why remove the `postId` context?

Reply:
I guess I removed it because either in the previous code or the current one, it's not used.
`get_the_ID(),` was used somehow. I have added it and replaced `get_the_ID()` to make it consistent with the condition check above.

Reply:
Oh I see— it's a better practice to keep it and fix get_methods_content (like [you did here](https://github.com/WordPress/wporg-developer/pull/475/commits/ee99c7aca5e3a30a27ece53fe693a28d51feee36) 👍🏻 ), since blocks _could_ be used in a query loop, where the "current post" (in context) would be different than the global post. It's not how we're using these blocks here, so it's not critical, but good to be in the habit of doing.

Reply:
Ah, I see. Thanks for clarifying this!